ide/posts/2020-05-22-tracing-plan.md

```diff
@@ -44,28 +45,41 @@ support over simple console logging.
 OpenTelemetry is a set of tools and APIs for monitoring applications in a
 variety of ways. We will be using its tracing portion which will allow us to
 view traces from ghcide in any opentelemetry-compatible viewer, like
-[Jaeger](https://jaegertracing.io/) or chrome's `about:tracing`. So, to address
-some confusion from the reddit comments on last week's post, this has nothing
-to do with telemetry, as used in e.g. Visual Studio Code or Windows. It is
-explicitly opt-in and does not transmit any data over the network.
+[Jaeger](https://jaegertracing.io/) or Google Chrome's `about:tracing`. So, to
+address some confusion from the reddit comments on last week's post: This has
+nothing to do with telemetry, as used in e.g. Visual Studio Code or Windows. It
+is explicitly opt-in and does not transmit any data over the network.
 
 Matthew Pickering (\@mpickering) has already started this effort. He has added
-initial support for tracing allowing us to get some basic traces. What I will
-be adding is more detail. The way shake works is that you issue *Actions* to be
-run, which can then request data, which based on some rules is calculated using
-other Actions, etc. We can currently only trace the initial actions, not any of
-their dependencies. For example, we now can see how long going to definition
-takes in its entirety, but not how much of that time was spent loading or
-parsing files. Adding this will give us a complete picture of what ghcide is
-doing.
-
-### That's great and all, but what do I get?
-
-Great question! Indeed, this is a developer-facing feature. We are not
-expecting users of ghcide to be looking at traces. We are hoping however it
-will guide us towards a faster ghcide by showing us what parts are worth
-improving. It should also make locating certain classes of bugs easier by
-showing us if Shake is doing what we are expecting it to.
+initial support for tracing allowing us to get some basic traces. You can see an
+example of what is possible below:
+
+<p align="center">
+![A ghcide trace viewed in chrome's about:tracing](/ide/images/trace-example.png){ width=800px }
```

Comment:
width=100% and put the picture above the <!-- more --> marker please

Reply:
What do you mean by above the marker?

Reply:
Ah I wrote `<!--more-->` but github ate the text because it's an html comment.. content above the marker appears on the front page. So just copy the image so it appears  after the intro paragraph but also here.

Reply:
Something like 4b56fe1?